conn/physic/units.go

```diff
@@ -352,6 +354,111 @@ const (
 	Fahrenheit      Temperature = 555555555 * NanoKelvin
 )
 
+// Power is a measurement of power stored as a nano watts.
+//
+// The highest representable value is 9.2GW.
+type Power int64
+
+// String returns the power formatted as a string in watts.
+func (p Power) String() string {
+	return nanoAsString(int64(p)) + "W"
+}
+
+const (
+	//Watt is unit of power J/s, kg⋅m²⋅s⁻³
+	NanoWatt  Power = 1
+	MicroWatt Power = 1000 * NanoWatt
+	MilliWatt Power = 1000 * MicroWatt
+	Watt      Power = 1000 * MilliWatt
+	KiloWatt  Power = 1000 * Watt
+	MegaWatt  Power = 1000 * KiloWatt
+	GigaWatt  Power = 1000 * MegaWatt
+)
+
+// Energy is a measurement of work stored as a nano joules.
+//
+// The highest representable value is 9.2GJ.
+type Energy int64
+
+// String returns the energy formatted as a string in Joules.
+func (e Energy) String() string {
+	return nanoAsString(int64(e)) + "J"
+}
+
+const (
+	//Joule is a unit of work. kg⋅m2⋅s⁻²
+	NanoJoule  Energy = 1
+	MicroJoule Energy = 1000 * NanoJoule
+	MilliJoule Energy = 1000 * MicroJoule
+	Joule      Energy = 1000 * MilliJoule
+	KiloJoule  Energy = 1000 * Joule
+	MegaJoule  Energy = 1000 * KiloJoule
+	GigaJoule  Energy = 1000 * MegaJoule
+)
+
+// Capacitance is a measurement of capacitance stored as a pico farad.
```

Comment:
I used ElectricResistance because I felt that Resistance was a bit ambiguous. Resistance can be measured also as Drag, Hardness and Thermal resistance.

I used ElectricCurrent and ElectricPotential to be coherent; so I feel that Capacitance may be worth having the same treatment, even if it's not ambiguous. Maybe I made an error, I'm still undecided.

Just thinking out loud here.

Reply:
I think having it as ElectricalCapacitance is more consistence with the current units even though there is no ambiguity and would cause less confusion than the other way; dropping the Electrical prefix.
Also thinking about it Flux probably should be LuminousFlux so as not to confuse lumens with webers or teslas in the future.